sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -648,6 +652,9 @@ template<typename T, typename PropertyListT>
 This property is an assertion by the application that the `annotated_ptr` has
 the given alignment, specified in bytes. The behavior is undefined if the
 pointer value does not have the indicated alignment.
+
+This property is passed to `annotated_ref` class when a `annotated_ptr` is
+deferenced with `operator*`.
```

Comment:
For this property the property should also be dropped for other operators (+, -, +=, ...).

Reply:
Yes will update all of them